app/controllers/admin/items_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::ItemsController < Admin::BaseController
+  before_action :require_admin
```

Comment:
I think if you're inheriting from the admin base controller, you don't need to have before_action here.